.gitignore

```diff
@@ -0,0 +1,6 @@
+Packages/
+Package.pins
+Package.resolved
+.xcodeproj
+.swiftpm
+.xcuserstate
```

Comment:
Please add `.build` directory as well (I forgot about that) and remove existing files from git index.